core/templates/pages/about-page/about-page.controller.ts

```diff
@@ -69,6 +77,15 @@ angular.module('oppia').directive('aboutPage', [
             return UrlInterpolationService.getStaticImageUrl(imagePath);
           };
           ctrl.$onInit = function() {
+            ctrl.allCredits = [];
+            var alphabetList = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ'.split('');
+            for (var i = 0; i < 26; i++) {
```

Comment:
should we use `i < alphabetList.length`? 
OR 
```
alphabetList.map((firstLetter) => {...})
```

Reply:
> should we use i < alphabetList.length?

Just to avoid random-looking number 26 (though it's not random :p )

Reply:
Yes, done!